docs/Single Sign On/okta.md

```diff
@@ -17,21 +18,57 @@ Login to your Okta account as an admin and navigate to `Applications -> Applicat
 
 In the dialog that opens, choose **OIDC - Open ID Connect** as the **Sign-in Method** and **Web Application** as the **Application Type.**
 
-On the next screen provide a name for the application. Under **Assignments -> Controlled Access**, select `Skip Group Assignment For Now`
-
 <img src="/img/docs/oidc-okta-1.png" alt="Creating New Okta OAuth App" height="75%" width="75%"/>
 
 Leave the **Application Callback URL** empty for now and Register the application.
 
-Note down the `client-id` and `client-secret` as those will be required to configure identity provider for Paralus.
+Based on who can access the Paralus app integration, select **Controlled access** options.
+
+Note down the `Client ID` and `Client Secret` as those will be required to configure identity provider for Paralus.
+
+### Add a custom claim for groups mapping
+
+Paralus comes with the in-built functionality for mapping Identity
+provider (IdP) groups to Paralus. This is useful to manage different
+Paralus access to the IdP users through IdP groups they are part
+of. It eases the access-management process for Parlaus
+administrators. See the limitation below.
```

Comment:
`See limitation below` - can be rephrased to 

_This however has a limitation. To map the groups correctly, the group names in Okta and Paralus must be the same. This way, the IdP users belonging to a group can access Paralus resources based on the permissions assigned to the group in Paralus. For example, you create a group called `DevTeam` in Paralus and the same group in Okta as well. Assign permissions to the Paralus group and add users to the Okta group. Now, when an IdP user belonging to the `DevTeam` group logs into Paralus using Okta, they will inherit the permissions set for the group. Read how to [manage users and groups in Okta](https://help.okta.com/oie/en-us/Content/Topics/users-groups-profiles/usgp-groups-main.htm)_